### DISCUSSION TITLE
Locking the interface also disables alt-arrow.

### DISCUSSION THREAD

**Discussion by debasish-raychawdhuri** [zellij-org/zellij#2471] [General]
Hi,
I have just installed zellij and using the default config. I just noticed that when I lock the interface, I can no long move between the panes using alt+arrow keys. Ideally, I would like to lock the interface, but would still like to be able move around.
Thanks.

**Comment by debasish-raychawdhuri** [comment]
Well It can be configured to do what I want, but I would say that should be default behaviour. Locking the interface should not stop people from moving around without changing the layout.

**Reply by Lockszmith-GH** [reply]
"Should" is a strong word.

As you said it's configurable. People have been using this without requesting this feature until now.
You want it a certain way, configure it, publish your configuration here and suggest it to other people. If this becomes popular it will surely be adopted.

You might not have meant it as a demand, but that's how it is read.

This is a project that is maintained by people on their free time. Demanding changes just because you prefer it this way brings negative vibes. Again - you might have not intended it to sound negative or 'strong', but it's important to keep this in mind.

Nobody appointed me guardian, nor am I a maintainer, I am just a user, like you. And I found this community and the maintainer to be awesome and helpful. But if you demand something, I doubt you'll get any response going forward.

Again, this might be a complete overreaction on my part, and if so, I apologize.

Let's help maintain these communities and not burn the maintainers out.

I'll post here some resources shared in another GitHub community I participate as a user:
* “Did You Miss My Comments Or What?” Toxicity In Open Source Discussions [summary](https://tylercipriani.com/blog/2023/01/15/entitlement-burnout-and-toxicity/) and [paper](https://cmustrudel.github.io/papers/osstoxicity22.pdf).
* [There is no "software supply chain"](https://iliana.fyi/blog/software-supply-chain/)
* [Entitlement in Open Source](https://mikemcquaid.com/entitlement-in-open-source/)
* [How ‘open’ should your open source be?](https://github.com/readme/featured/how-open-is-open-source) on the GitHub blog

**Comment by debasish-raychawdhuri** [comment]
I did not mean to demand anything. It was just a suggestion.
